### PR TITLE
Added adm command

### PR DESCRIPTION
**Pull request by vedangj044**
Initial RFC for `drg adm` commands

### CODE REVIEW

File: active/0009-drg-admin.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+### Adding members
+
+`drg adm add --app <app_id> <username> --roles <reader/manager/admin>`
+
+This command helps assign roles to another user. Here we would need to pass in the user id of the other user and the permission we would like to give him.
+
+`drg adm list --app <app_id>`
+
+This command prints all the members with access to this application and their roles.
```
Comment (jbtrystram) [inline active/0009-drg-admin.md:21]: I suggest that this list is simply the object as YAML

File: active/0009-drg-admin.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+`drg adm transfer --app <app_id> -d`
+
+By adding an -d or --delete flag we can cancel the ownership transfer process.
+
+
+`drg adm transfer --app <app_id> -a`
+
+By adding an -a or --accept flag, the user can accept the ownership of an application.
```
Comment (jbtrystram) [inline active/0009-drg-admin.md:47]: drg admin transfer start 
drg admin transfer accept
drg admin transfer cancel

More intuitive IMO

File: active/0009-drg-admin.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Drogue Cloud also allows an application to have multiple owners, and different people to have different roles in the same application. Here, we try to document how drg can be used for this purpose.
+
+## Commands
+
+We love to group commands based on their functionality so let's keep the subcommand for all the administration work to - drg `adm`. 
+
+### Adding members
+
+`drg adm add --app <app_id> <username> --roles <reader/manager/admin>`
```
Comment (jbtrystram) [inline active/0009-drg-admin.md:15]: drg admin roles add <reader/manager> <userId>
drg admin roles edit -> same as device edit

File: active/0009-drg-admin.md (hunk 2 above)
Comment (ctron) [reply active/0009-drg-admin.md:47]: I would prefer not to use `--app`, as this is a mandatory argument.

File: active/0009-drg-admin.md (hunk 3 above)
Comment (ctron) [reply active/0009-drg-admin.md:15]: The member can have only one role. Not sure that will change in the future. So you are not adding a role, but a user. Maybe the following makes more sense:

~~~
drg admin member add <user> <reader|manager|…>
~~~

There is also the "anonymous" user, which is technically `""` … not sure how to handle this with the command line.

File: active/0009-drg-admin.md (hunk 1 above)
Comment (jbtrystram) [reply active/0009-drg-admin.md]: That is up to you if you want to implement it :)

File: active/0009-drg-admin.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Overview
+
+In Drogue cloud, Applications scope all elements that belong together, like devices, data, and integrations. In recent improvements to DRG - we are using applications as a root of trust to authenticate devices. 
+
+Drogue Cloud also allows an application to have multiple owners, and different people to have different roles in the same application. Here, we try to document how drg can be used for this purpose.
+
+## Commands
+
+We love to group commands based on their functionality so let's keep the subcommand for all the administration work to - drg `adm`.
```
Comment (jbtrystram) [inline active/0009-drg-admin.md:11]: why a 3 letters subcomand ? `drg admin` works fine IMO

**Review by jbtrystram — APPROVED** [review]
LGTM. Let's go with that :)

**Pull request merged**